packages/frameworks/vue/src/presence/presence.props.ts

```diff
@@ -1,21 +1,12 @@
-import type { Context } from '@zag-js/presence'
-import type { PropType } from 'vue'
 import { declareEmits } from '../utils'
-import type { UsePresenceProps } from './use-presence'
 
 export const props = {
   present: {
-    type: Boolean as PropType<Context['present']>,
+    type: Boolean,
     default: undefined,
   },
-  lazyMount: {
-    type: Boolean as PropType<UsePresenceProps['lazyMount']>,
-    default: false,
-  },
-  unmountOnExit: {
-    type: Boolean as PropType<UsePresenceProps['unmountOnExit']>,
-    default: false,
-  },
+  lazyMount: Boolean,
+  unmountOnExit: Boolean,
```

Comment:
`as PropType...` should be here, simply because `x.props.ts` is a generated (`pnpm scripts vue:props`) file with good typing. It tells you what is bad in a more clean way.
I tried 
![image](https://github.com/chakra-ui/ark/assets/6079265/b6c915ad-9892-4aac-96fb-1becb6b95635)
but this is not clean as the old method.
![image](https://github.com/chakra-ui/ark/assets/6079265/ee49ab9a-7052-490d-9c34-69c8cd7c8b4f)

With the old method:
![image](https://github.com/chakra-ui/ark/assets/6079265/da4e975e-3862-4ba6-992d-6fc3df786413)